packages/ChangesReloaded-Core.package/CRBrowser.class/instance/buildButtonsWith..st

```diff
@@ -0,0 +1,13 @@
+building
+buildButtonsWith: aBuilder
```

Comment:
about the names of all of those `build...With:` methods: when we were working on our part, we didn't think the naming fits to what actually happens there. This is why we named that window method `createWindowSpecWith:`. In there, nothing is built, and especially not the widgets themselves. Instead, a _Spec_ of a widget is created. The actual building happens only in the `buildWith:` method (or rather in the `build:` method that gets called there). So my suggestion would be `createButtonSpecsWith:`, other methods analogously

Reply:
After some discussion, we'll probably name these kinds of methods `buildButtonSpecsWith:`

Reply:
`buildButtonPanelSpecWith:`